Bulldozer/packages.config

```diff
@@ -5,11 +5,10 @@
   <package id="CacheManager.StackExchange.Redis" version="1.1.2" targetFramework="net472" />
   <package id="CacheManager.SystemRuntimeCaching" version="1.1.2" targetFramework="net472" />
   <package id="Costura.Fody" version="3.1.4" targetFramework="net472" />
-  <package id="DDay.iCal" version="1.0.2.575" targetFramework="net472" />
```

Comment:
Did you actually determine it was necessary to remove DDay.iCal or could this possibly cause issues?

Reply:
@nateh777 I have not encountered any issues. I ran a full csv and binary import on this current code with no build errors, execution errors, or RockDB ErrorLog errors.

Reply:
Okay, let's hope it isn't required for some one off thing that leads us down a rabbit hole to fix when we do a migration. We'll have to just remember we removed it if we have an issue related to ical or schedules.